wdn/templates_4.0/scripts/navigation.js

```diff
@@ -44,6 +45,72 @@ define(['jquery', 'wdn', 'modernizr', 'require'], function($, WDN, Modernizr, re
 		return setTimeout(callback, 0);
 	};
 
+	var _now = Date.now || function() {
+		return new Date().getTime();
+	};
+
+	var throttle = function(func, wait, options) {
+		var context, args, result;
+		var timeout = null;
+		var previous = 0;
+		if (!options) options = {};
+		var later = function() {
```

Comment:
This looks good, but I have two thoughts.  These functions are pretty hard to follow, especially if you are new to them and what they are supposed to do.  I think some comments could really add to the readability.  For example, what is the purpose of `previous` and what sort of values should it contain?  what is `later` used for?

I could also see an unit tests be very helpful for functions like these.